CHANGELOG.md

```diff
@@ -3,23 +3,63 @@
 ## v2.0.0-preview.1
 ### Wednesday, January 23, 2019
 
+#### Preview builds of the PowerShell extension are now available in VSCode
+
+We are excited to announce the PowerShell Preview extension in the VSCode marketplace!
+The PowerShell Preview extension allows users on Windows PowerShell 5.1 and newer to get and test the latest updates to the PowerShell extension and comes with some exciting features.
+
+The PowerShell Preview extension is a substitute for the PowerShell extension so both the PowerShell extension and the PowerShell Preview extension should not be enabled at the same time.
+
+By having a preview channel, which only supports Windows Powershell 5.1 and newer, in addition to our existing stable channel, we can get new features out faster.
```

Comment:
I think this line is trying to say two things, which are:

> With the preview channel, we will be able to get feedback on recent additions to the extension, allowing us to develop better features without compromising stability.
>
> NOTE: As of version 2.0.0, the PowerShell extension no longer supports PowerShell 3 or 4. This is a requirement of the PSReadLine integration, and is discussed at length [here](https://github.com/PowerShell/vscode-powershell/issues/1310).

Reply:
can you also give this feedback in the Blog post

Reply:
fixed